stress/common/src/main/java/alluxio/stress/jobservice/JobServiceBenchTaskResultStatistics.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress.jobservice;
+
+import alluxio.Constants;
+import alluxio.stress.StressConstants;
+import alluxio.stress.common.SummaryStatistics;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import org.HdrHistogram.Histogram;
+
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.zip.DataFormatException;
+
+/**
+ * Statistics class that is used in {@link JobServiceBenchTaskResult}.
+ */
+public class JobServiceBenchTaskResultStatistics {
```

Comment:
this class seems very similar to `MasterBenchTaskResultStatistics`, any ways we could refactor here? what makes these two different?

Reply:
yeah it is, the only difference is the max response time buckets, we definitely can do small refactoring as we discussed before. I would personally prefer to get rid of this class and just put all the stuff into TaskResult class and init the TaskResult class with a parameter for the max response time buckets instead of using a constant.

Reply:
Let's add a TODO and do the refactoring in a separate PR?